src/components/SpruceForm/Widgets/DistroDropdown.tsx

```diff
@@ -0,0 +1,128 @@
+import styled from "@emotion/styled";
```

Comment:
This feels like to specific of a component to be a SpruceForm component. SpruceForm is meant to be generic and reusable to create forms anywhere on spruce we shouldn't have a widget that is specific to one form. I think this should just be a generic SearchableDropdown widget.